x-pack/plugins/observability_ai_assistant/server/functions/get_dataset_info.ts

```diff
@@ -115,7 +115,7 @@ export function registerGetDatasetInfoFunction({
       const relevantFields = await Promise.all(
         chunk(fieldNames, 500).map(async (fieldsInChunk) => {
           const chunkResponse$ = (
-            await client.chat({
+            await client.chat('get_relevent_dataset_names', {
```

Comment:
Would it make sense to tag all of these "internal" calls as part of a function? So it's easy to filter by the chat calls we make from a function?

Reply:
it happens automatically because its parent span is a function call (`execute_function ...`)